source/layout.html

```diff
@@ -52,7 +52,7 @@
         <ul>
           <li{% if page.full_url contains '/products' or page.full_url contains '/product/' or page.full_url contains '/category/' or page.full_url contains '/artist/' %} class="selected"{% endif %}><a href="/products">{{ pages.products.name }}</a></li>
           <li{% if page.full_url contains '/cart' %} class="selected"{% endif %}><a href="/cart">Cart</a></li>
-          <li class="more-link"><a href="#footer" title="See more">More</a></li>
+          <li class="more-link"><a href="#footer" title="See more" aria-label="View additional navigation">More</a></li>

```

Comment:
Do you think that title should change too to match the aria-label?

Reply:
@cella Yup, that's a good idea.